spot_rl_experiments/configs/config.yaml

```diff
@@ -136,7 +159,8 @@ INITIAL_ARM_JOINT_ANGLES_SEMANTIC_PLACE_LEFT_HAND: [0, -180, 180, 0, 0, -90] # T
 INITIAL_ARM_JOINT_ANGLES_SEMANTIC_PLACE_TOP_DOWN: [-0.91, -92.87, 91.21, 0, 90.01, 0] # The initial orientation of the arm for top down grasping
 # The old gaze ready angle: [0, -170, 120, 0, 75, 0]
 GAZE_ARM_JOINT_ANGLES: [0, -160, 100, 0, 75, 0] #[0, -160, 100, 0, 90, 0]
-SEMANTIC_PLACE_ARM_JOINT_ANGLES: [0, -125, 80, 0, 85, 0]
+SEMANTIC_PLACE_ARM_JOINT_ANGLES: [0, -120, 60, 0, 88, 0]
```

Comment:
Why are we changing this value? We should restore the old value and remove the commented out line below

Reply:
Restore the old value